tests/unibit-functions.test.js

```diff
@@ -0,0 +1,55 @@
+const Unibit = require('../src/unibit');
+const pageContext = require('./mocks/pageContext.mock.json')
+const unibit = new Unibit();
+
+describe('getPage', ()=>{
+    it('gets a page by url', () => {
+        let page = unibit.getPage(pageContext, 'about');
+        expect(page).toBe(pageContext.site.pages[0]);
+    });
```

Comment:
Don't use huge file just to check correct page selection.
Use data to test only what is needed:
And don't assume indexes in test data but use references
```
const expectedPage = {url:"...",...smallestDataNeededForTheTest},
const pages = [
  {...smallestDataNeededForTheTest},
  expectedPage,
  {...smallestDataNeededForTheTest}
]
const page = unibit.getPage({site:{pages:pages, 'about');
expect(page).toBe(expectedPage);
```